dll/win32/shell32/lang/es-ES.rc

```diff
@@ -747,10 +753,10 @@ BEGIN
     IDS_ADMINISTRATIVETOOLS "Herramientas administrativas"
 
     /* special folders descriptions */
-    IDS_ADMINISTRATIVETOOLS_DESCRIPTION "Configure administrative settings for your computer."
-    IDS_FOLDER_OPTIONS_DESCRIPTION "Customize the display of files and folders and change file associations."
-    IDS_FONTS_DESCRIPTION "Add, change, and manage fonts on your computer."
-    IDS_PRINTERS_DESCRIPTION "Shows installed printers and fax printers and helps you add new ones."
+    IDS_ADMINISTRATIVETOOLS_DESCRIPTION "Configuraciones administrativas para su computadora."
```

Comment:
In Spain, we call the computer "ordenador". Windows normally uses "PC" to avoid that conflict, such as in "Mi PC" in place of "My Computer". :P

Reply:
I was thinking this also, but in other hispanic countries than Spain, the word "ordenador" means anything than personal computer, so I believe that in aim of a widespread translation, the correct word should be "computador(a)"